### ISSUE TITLE
04 Jan 2024 - A11y Theme Builder Meeting Minutes

### ISSUE THREAD

**Issue by aaronreed708** [finos/a11y-theme-builder#763]
## Date
20240104 - 9:15am EST

## Zoom info
- Meeting link: https://zoom.us/j/91013899295?pwd=Qnh5UVN1aUZ4ak9tODgvMVNLNUg5Zz09
- Meeting id: 910 1389 9295
- Password: 123225
- Call-in: +1 929 205 6099 US (New York) (Check calendar for more options)

## Untracked attendees
- Fullname, Affiliation, (optional) GitHub username
- ...

## Meeting notices
- FINOS **Project leads** are responsible for observing the FINOS guidelines for [running project meetings](https://community.finos.org/docs/governance/meeting-procedures/). Project maintainers can find additional resources in the [FINOS Maintainers Cheatsheet](https://community.finos.org/docs/finos-maintainers-cheatsheet).

- **All participants** in FINOS project meetings are subject to the [LF Antitrust Policy](https://www.linuxfoundation.org/antitrust-policy/), the [FINOS Community Code of Conduct](https://community.finos.org/docs/governance/code-of-conduct) and all other [FINOS policies](https://community.finos.org/docs/governance/#policies). 

- FINOS meetings involve participation by industry competitors, and it is the intention of FINOS and the Linux Foundation to conduct all of its activities in accordance with applicable antitrust and competition laws. It is therefore extremely important that attendees adhere to meeting agendas, and be aware of, and not participate in, any activities that are prohibited under applicable US state, federal or foreign antitrust and competition laws. Please contact legal@finos.org with any questions.

- FINOS project meetings may be recorded for use solely by the FINOS team for administration purposes. In very limited instances, and with explicit approval, recordings may be made more widely available.

## Agenda
- [x] Convene & roll call (5mins)
- [x] Display [FINOS Antitrust Policy summary slide](https://community.finos.org/Compliance-Slides/Antitrust-Compliance-Slide.pdf) 
- [x] Review Meeting Notices (see above)
- [x] Approve past meeting minutes
- [x] Happy New Year!  The adventure continues!
- [x] Help requested defining 2024 Roadmap (#762)
- [x] Awaiting Feedback items addressed by Lise and Aaron
- [x] AOB, Q&A & Adjourn (5mins)

## Minutes

- After well wishes for a Happy New Year, we covered the agenda items
- Aaron encouraged all to update #762 with key events/dates, requirements, suggested prioritization and general vision of how we'd like to progress the project in 2024.  We will schedule a separate meeting to discuss the 2024 roadmap in a couple of weeks.
    - The Community Working Group should be sure to weigh in here
- Aaron summarized meeting with Lise yesterday, discussing various issues that were blocked waiting for clarifications
    - all "Awaiting Feedback" issues now have clarifications and the "Awaiting Feedback" labels have been removed from them.  All of these issues should now be unblocked.  Please contact @aaronreed708 if further clarifications are necessary.
- After going through current status for the "in progress" issues, we grouped the color blindness/charting issues into a group in the High Priority column
    - Lise would love to have charting and color blindness implemented in time for the next conference but we still need to size everything to see if that is feasible
- Lise suggested that we combine some of our Design Thinking issues together to come up with a cohesive UX so that all of the new UI that we are designing is consistent.
    - we don't want some things to have dropdowns and others to use switches to achieve the same goals, for example, just because their design thinking sessions were held weeks apart by different facilitators
    - Aaron to open an issue to combine the efforts where they make sense
- Lise noticed that some issues were missing from the board.  Aaron to investigate.

## Action Items
- [x] Aaron to investigate issues missing from the board
- [x] Aaron to update #686
- [ ] Aaron to open issue to combine Design Thinking sessions
- [x] Aaron and Lise to meet to resolve development issues with Chart.js
- [ ] Aaron needs to re-prioritize (and re-order) the high and low priority column issues to make sure that they are accurate

**Comment by codyzu** [comment]
Cody Zuschlag, NearForm, @codyzu

**Comment by evangk6** [comment]
Evan K - DFS

**Comment by aaronreed708** [comment]
Aaron R - DFS
Lise N - DFS

**Comment by aaronreed708** [comment]
Approved in today's call
